### ISSUE TITLE
add log to debug invalid session

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2652]

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.35.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.35.0...4.35.1) (2022-04-12)

- Release 4.35.1 [`c1e76b8`](https://github.com/open-wa/wa-automate-nodejs/commit/c1e76b8744ba98ac49ee5f17407feafe48a35c61)
- 🔊 debug logging for invalid session #2652 [`03e85f7`](https://github.com/open-wa/wa-automate-nodejs/commit/03e85f71ca7badcc0626b6acdec063cb3182525a)
- updated types-only package [`1753190`](https://github.com/open-wa/wa-automate-nodejs/commit/175319071e4c1eaa503c1bfa7e18f36ffefd29bc)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.35.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.35.1...4.35.2) (2022-04-12)

- Release 4.35.2 [`9634914`](https://github.com/open-wa/wa-automate-nodejs/commit/9634914f6fb1e66d7eb9d0c31e8f0f91a471450f)
- 🔊 fix invalid session logs #2652 [`c6e119d`](https://github.com/open-wa/wa-automate-nodejs/commit/c6e119d7cba041e8024ff95098aa9a4e9699c2b8)
- updated types-only package [`329017b`](https://github.com/open-wa/wa-automate-nodejs/commit/329017b072a9e81b84b1628eb778b4096216313b)
